### PR TITLE
Fix lints and update golanci-lint config

### PR DESCRIPTION
**Pull request by cpanato**
- Fix lints and update golanci-lint config

### CODE REVIEW

File: cmd/update-rules/main.go [hunk 1]
```diff
@@ -126,7 +128,7 @@ func UpdateRules(rules *config.RepositoryRules, branch, goVer string) {
 		var newBranchRule config.BranchRule
 		// find the mainBranch rules
 		for _, br := range r.Branches {
-			if br.Name == "master" {
+			if br.Name == MainBranchName {
```
Comment (cpanato) [inline cmd/update-rules/main.go:131]: As a follow up, should we prepare this to be both `master` and `main` to be ready when changing k/k branch name to `main`?

Comment (dims) [reply cmd/update-rules/main.go:131]: sounds good as follow up PR.

**Pull request merged**